.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### What?
```

Comment:
```suggestion
## What?
```
Just personal preference from me, but I like the h2 with a underline 😃

Reply:
Haha OK, I don't have a strong preference so I can change it!